app/browser/tabs.js

```diff
@@ -302,7 +302,8 @@ const fixDisplayURL = (navigationEntry, controller) => {
     navigationEntry.virtualURL = getSourceAboutUrl(navigationEntry.virtualURL)
   }
 
-  if (isIntermediateAboutPage(navigationEntry.virtualURL)) {
+  if (isIntermediateAboutPage(navigationEntry.virtualURL) &&
+    navigationEntry.virtualURL.indexOf('about:safebrowsing#') === -1) {
```

Comment:
if this is an issue for about:safebrowsing, shouldn't it also be an issue for about:error, about:certerror, etc?